server/src/controller/adminController.js

```diff
@@ -0,0 +1,89 @@
+import {
+  getAllItem,
+  addItem,
+  toggleActiveStatus,
+  updateItem,
+  getMessage,
+  deleteMessage,
+} from "../models/adminModel.js";
+
+export const getItemController = async (req, res) => {
+  try {
+    const item = await getAllItem();
+    return res.json(item);
+  } catch (error) {
+    console.log(error);
+    return res.status(400).json({ error });
+  }
+};
+
+// Post request
+export const addItemController = async (req, res) => {
+  const categoryId = Number.parseInt(req.body.categoryId, 10);
```

Comment:
This can be shortened to:

```suggestion
  const categoryId = Number(req.body.categoryId);
```